src/components/responsive-hero-image.js

```diff
@@ -18,9 +19,23 @@ import omit from '@babel/runtime/helpers/objectWithoutProperties';
  * @category Images
  */
 export function ResponsiveHeroImage(props) {
-  return React.createElement(ResponsiveImage, Object.assign({
-    slug: props.story["hero-image-s3-key"],
-    metadata: props.story["hero-image-metadata"],
-    alt: props.story["headline"]
-  }, omit(props, ['story'])));
+  const storyAlternateData = get(props, ["story", "alternative", "home", "default"], {});
+  const { headline: altHeadline } = storyAlternateData;
+  const { "hero-image-s3-key": altSlug, "hero-image-metadata": altMetadata } = get(storyAlternateData, ["hero-image"], {});
+
+  const slug = get(props, ["story", "hero-image-s3-key"], altSlug);
+  const metadata = get(props, ["story", "hero-image-metadata"], altMetadata);
+  const alternateText = get(props, ["story", "headline"], altHeadline);
```

Comment:
what will happen if a story has hero-image-s3-key but hero-image-metadata is absent? This logic will pick the wrong combination